cutil/cutil.go

```diff
@@ -826,3 +826,9 @@ func GetCertificateVersion(certFilePath string) string {
 	}
 	return version
 }
+
+func CloseFileLogError(f *os.File) {
+	if err := f.Close(); err != nil {
```

Comment:
Could you add a check of f == nil to prevent someone pass in a nil file handle unexpectly?

Reply:
Done